07-routing-gossip.md

```diff
@@ -987,7 +987,7 @@ A node:
 #### Requirements
 
 A node:
-  - if a channel's oldest `channel_update`s `timestamp` is older than two weeks
+  - if a channel's newest `channel_update`s `timestamp` is older than two weeks
```

Comment:
nit: we don't use "newest" anywhere in the repo, but it seems like we use "latest" for that:
```suggestion
  - if a channel's latest `channel_update`s `timestamp` is older than two weeks
```